frontend/package.json

```diff
@@ -0,0 +1,115 @@
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.17.0",
+    "express": "^4.17.1",
+    "helmet": "^3.21.2",
+    "humps": "^2.0.1",
+    "imagemin-gifsicle": "^6.0.1",
+    "imagemin-mozjpeg": "^8.0.0",
+    "imagemin-optipng": "^7.1.0",
+    "imagemin-pngquant": "^8.0.0",
+    "imagemin-svgo": "^7.0.0",
+    "immutable": "^4.0.0-rc.12",
+    "inline-worker": "^1.1.0",
+    "next": "^9.1.4",
+    "next-optimized-images": "^2.5.4",
+    "next-redux-wrapper": "^4.0.1",
+    "next-routes": "^1.4.2",
+    "node-sass": "^4.13.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.12.0",
+    "react-bootstrap": "^1.0.0-beta.16",
+    "react-dom": "^16.12.0",
+    "react-helmet": "^5.2.1",
+    "react-hook-form": "^3.28.11",
+    "react-markdown": "^4.2.2",
+    "react-redux": "^7.1.3",
+    "react-slick": "^0.25.2",
+    "redux": "^4.0.4",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.3.0",
+    "slick-carousel": "^1.8.1"
+  },
+  "name": "almond-frontend",
+  "version": "0.0.1",
+  "main": "server/index.js",
+  "devDependencies": {
+    "@babel/core": "^7.1.0",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-eslint": "^10.0.3",
+    "babel-jest": "^24.9.0",
+    "babel-plugin-inline-dotenv": "^1.5.0",
+    "babel-plugin-module-resolver": "^3.2.0",
+    "eslint": "^6.7.2",
+    "file-loader": "^5.0.2",
+    "image-webpack-loader": "^6.0.0",
+    "jest": "^24.9.0",
+    "lint-staged": "^9.5.0",
+    "nodemon": "^2.0.1",
+    "pre-commit": "^1.2.2",
+    "raw-loader": "^4.0.0",
+    "react-test-renderer": "^16.12.0",
+    "rimraf": "^3.0.0",
+    "snazzy": "^8.0.0",
+    "standard": "^14.3.1",
+    "url-loader": "^3.0.0",
+    "webpack-bundle-analyzer": "^3.6.0"
+  },
+  "scripts": {
+    "lint-staged": "lint-staged",
+    "build": "NODE_ENV=production next build",
+    "test:ci": "jest --maxWorkers=2 --ci",
+    "test": "jest --watch",
+    "start": "nodemon -w server server/index.js",
+    "serve": "NODE_ENV=production node server/index.js",
+    "clean": "rimraf node_modules/.cache .next",
+    "lint": "standard --verbose | snazzy",
+    "now-start": "yarn serve",
+    "lint:fix": "standard --fix --verbose | snazzy"
+  },
+  "pre-commit": "lint-staged",
+  "lint-staged": {
+    "*.js": [
+      "yarn lint:fix",
+      "git add"
+    ]
+  },
+  "standard": {
+    "parser": "babel-eslint",
+    "env": [
+      "jest"
+    ]
+  },
+  "engines": {
+    "node": ">= 8.12"
+  },
+  "jest": {
+    "modulePaths": [
+      "./src",
+      "./src/test"
+    ],
+    "setupFilesAfterEnv": [
+      "@testing-library/jest-dom/extend-expect"
+    ]
+  },
+  "repository": "https://github.com/stanford-oval/almond-cloud",
+  "bugs": "https://github.com/stanford-oval/almond-cloud/issues",
+  "author": "Euirim Choi <euirim@stanford.edu> (https://github.com/euirim)",
+  "license": "MIT",
+  "description": "Frontend of Almond's website.",
+  "private": true
```

Comment:
I understand this was done in a separate branch, but this need to be merged with the existing package.json - and individual changes justified and approved. There's a lot of workflow changes here that need to be justified. Also, a ton of dependencies, which is ok by itself, but it's really a lot.

Reply:
Yep, resolved on Slack.